pywhat/Data/regex.json

```
@@ -1309,6 +1309,23 @@
          ]
       }
    },
+   {
+      "Name": "UPS Tracking Number",
+      "Regex": "^(1Z[0-9A-Z]{6}[0-9]{2}[0-9]{8})$",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 1,
+      "URL": "https://www.ups.com/track?tracknum=",
+      "Tags": [
+         "Tracking"
+      ],
+      "Examples": {
+         "Valid": [
+            "1Z123ABC0012345678"
```

More examples taken from https://www.quora.com/What-is-an-example-of-a-UPS-tracking-number

```suggestion
            "1Z123ABC0012345678",
            "1Z9999999999999999",
            "1Z999AA10123456784"
```